db/ddl.sql

```diff
@@ -116,35 +98,35 @@ GRANT ALL ON TABLE region TO "tttAdmin";
 
 
 /********************************************
-  person
+  employee
 ********************************************/
 
-DROP TABLE IF EXISTS person CASCADE;
+DROP TABLE IF EXISTS employee CASCADE;
 
-CREATE TABLE person (
+CREATE TABLE employee (
   "id" INT PRIMARY KEY,
   "email" VARCHAR(255) NULL,
-  "name" VARCHAR(255) NULL,
+  "name" VARCHAR(255) NULL
 );
```

Comment:
Just catching this but are `employee` and `account` tables distinct from one another / representing different things?

Reply:
That's a good consideration. I suppose it would depend on how we view the system as a whole. I think if we view account holders as separate from the employees, then it's fair to keep the two tables separate, despite the duplicated fields.

However, if we think that all employees are account holders with the capability to sign in, then yeah - I think we should definitely just house it all in one table. I'm okay with it either way.